package.json

```diff
@@ -6,15 +6,15 @@
     "start": "ng serve --watch",
     "build": "ng build",
     "build-prod": "ng build --configuration production",
-    "test": "ng test --no-watch --no-progress --browsers ChromeHeadless --code-coverage",
+    "test": "ng test --no-progress --no-watch --browsers ChromeHeadless --code-coverage",
     "test-firefox": "ng test --no-progress --browsers Firefox --code-coverage",
     "lint": "eslint . --ext .ts",
     "lint-and-fix": "eslint . --ext .ts --fix",
     "stylelint": "stylelint '**/*.scss'",
     "stylelint-fix": "stylelint '**/*.scss' --fix",
     "prettier": "prettier --write '**/*.{ts,scss,js,html,spec.ts}'",
     "prettier-check": "prettier --check '**/*.{ts,scss,js,html,spec.ts}'",
-    "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
+    "postinstall": "",
```

Comment:
Garbage?

Reply:
Yes, this is not necessary for the new version.